tests/js/spec/views/eventsV2/table/cellAction.spec.jsx

```diff
@@ -232,6 +239,20 @@ describe('Discover -> CellAction', function() {
       wrapper.find('MenuButton').simulate('click');
       expect(wrapper.find('button[data-test-id="release"]').exists()).toBeFalsy();
     });
+
+    it('use custom renderer for key transaction', function() {
```

Comment:
Maybe also mock out the save call and check that clicking save updates the state?

Reply:
I added tests to assert that the call back gets passed the appropriate values rather than peaking into the states directly.